Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSource.java

```diff
@@ -108,26 +109,20 @@ private CardImageUrls innerGenerateURL(CardDownloadData card, boolean isToken) {
         // basic cards by api call (redirect to img link)
         // example: https://api.scryfall.com/cards/xln/121/en?format=image
         if (baseUrl == null) {
+            String cn= card.getCollectorId() ;
+            if( cn.endsWith( "*" )) cn= cn.substring(0 , cn.length() -1) +"★" ;
+            else if( cn.endsWith( "+" )) cn= cn.substring(0 , cn.length() -1) +"†" ;
+            else if( cn.endsWith( "Ph" )) cn= cn.substring(0 , cn.length() -2) +"Φ" ;
```

Comment:
If you use IntelliJ IDEA then try to press CTRL+ALT+L in commit dialog or in editor to auto-format source code. Do not use auto-format in other IDEs. Default java formating from IntelliJ IDEA is standard for all xmage source code except GUI related files from NetBeans.

Reply:
Had to guess with regards to the formatting you want because syncing the fork seems to have messed up IntelliJ (This was the first time I used it.) So used Notepad++ to make these changes.